frontend/src/lib/components/ActivityLog/ActivityLog.tsx

```diff
@@ -4,9 +4,9 @@ import { TZLabel } from 'lib/components/TimezoneAware'
 import { useValues } from 'kea'
 import './ActivityLog.scss'
 import { activityLogLogic } from 'lib/components/ActivityLog/activityLogLogic'
-import { Skeleton } from 'antd'
 import { ActivityScope, Describer } from 'lib/components/ActivityLog/humanizeActivity'
 import { PaginationControl, usePagination } from 'lib/components/PaginationControl'
+import { Skeleton } from '../Skeleton/Skeleton'
```

Comment:
Let's do `export { Skeleton } from './Skeleton` in `/Skeleton/index.tsx`, so that there's no need for `*/Skeleton/Skeleton`